src/regex/pcre2.cr

```diff
@@ -6,9 +6,23 @@ module Regex::PCRE2
   @re : LibPCRE2::Code*
   @jit : Bool
 
+  def self.version : String
```

Comment:
```suggestion
  # :nodoc:
  def self.version : String
```

Reply:
At least, it's not clear to me that we want to expose the two. Also, wouldn't it make sense to make these lazy properties?

Reply:
The entire module is `nodoc`, no need to put that on the individual methods.

Caching certainly makes sense, though.